commit-testing-to-current.sh

```diff
@@ -1,18 +1,17 @@
 #!/bin/sh
 
+set -eu
+if ! { [ $# -eq 3 ] && [ -z "$1" ]; }; then
+    echo "Usage: $0 <path-to-current-testing-repo-snapshot> [\"<component list>\" [<release-name>]]"
```

Comment:
Since you've made the second and third arguments mandatory, update usage info too.